SampleApp/SampleApp/ViewController.m

```diff
@@ -20,6 +21,11 @@ - (void)viewDidLoad {
     // Do any additional setup after loading the view, typically from a nib.
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    [self presentViewController:[SampleAlertViewController new] animated:YES completion:nil];
+}
+
 - (IBAction)registerUser:(id)sender {
     // "Register User" button clicked
     [PlacedAgent registerUser];
```

Comment:
We need to make sure the user has accepted the SampleAlertViewController before registering the user right?